test/integration/instance.test.js

```diff
@@ -1759,6 +1759,38 @@ describe(Support.getTestDialectTeaser('Instance'), function() {
         });
       });
     });
+
+    it('delete a record of multiple primary keys table', function() {
+      var MultiPrimary = this.sequelize.define('MultiPrimary', {
+        bilibili: {
+          type: Support.Sequelize.CHAR(2),
+          primaryKey: true
+        },
+
+        guruguru: {
+          type: Support.Sequelize.CHAR(2),
+          primaryKey: true
+        }
+      });
+
+      return MultiPrimary.sync({ force: true }).then(function() {
+        return MultiPrimary.create({ bilibili: 'bl', guruguru: 'gu' }).then(function() {
+          return MultiPrimary.create({ bilibili: 'bl', guruguru: 'ru' }).then(function(m2) {
+            return MultiPrimary.findAll().then(function(ms) {
+              expect(ms.length).to.equal(2);
+              return m2.destroy({
```

Comment:
We prefer not to test the SQL in integration tests (older tests that do that are slowly being ripped out).
Can you please refactor the test to check the expected result of a findAll after the destroy?